### PR TITLE
docs: Blog post on Compose guestbook application

### LINKED ISSUE #3345: Demo Scenario for Compose
### Goals:
Blog post + Demo materials on https://github.com/redhat-developer/podman-desktop-demo

### Goals of the demo:
- Podman and Podman Desktop does not break existing developer flows which rely on Compose
- It is easy to configure
- Re-emphasis that we are providing a nice UI for working with compose

### Sample code
https://github.com/redis-developer/beats-by-redis

### Features we want to show
Being able to install docker compose system wide / from a fresh install without having to go find the binary?
Standard features: Start/Stop/Restart/Delete/Logs/Inspect/Terminal/Summary

### High Level Flow
**Initial steps:**
- Starting from an existing application with an “docker compose up” from the IDE
- → Show that it works, that in the terminal I can see the image built and started
- Jump into Podman Destkop
- → Show the UI with the list of containers, I can see my group of compose containers
- → I can inspect, we do a tour of the features, logs, ssh, start stop delete

**Second steps:**
- What if I want to go to kubernetes, because that’s the platform I want to deploy my app onto
- → Show the kubernetes YAML tabs, it helps getting started
- And then, what if you were using kubernetes yamls the way you use compose
- → we switch the source code from compose to yamls (that means, we prepared the yamls before)
- → we explain the yamls, how they are defined, how they link to dockerfile/containerfile
- → then, we do a “play kube” from the ide, just with podman
- → we show it works
- → we show we can deploy it onto kubernetes with a kind cluster.

### PR DESCRIPTION
**Pull request by cdrage**
docs: Blog post on Compose guestbook application

### What does this PR do?

* Adds a blog post on creating a multi-container application by using
Compose and Podman Desktop.
* This application is a "guestbook" example where the front end is a
  guestbook-like application you can leave your name on.
* Multi-container database which is a Redis master and Redis replica(s).

### Screenshot/screencast of this PR

See Netlify / preview markdown.

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/3345

### How to test this PR?

Read the tutorial and go through the example :)

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
NOTES:

For the compose YAML steps instead do:

```sh
git clone https://github.com/redhat-developer/podman-desktop-demo
cd podman-desktop-demo/compose
podman compose up -d
```

I propose we get this demo into its own repo as well.

Before we merge I propose we create a repo at https://github.com/redhat-developer/podman-desktop-compose-example

So we avoid pulling in the rest of the demo applications from the redhat-developer/podman-desktop-demo repo.

The compose example also brings in a vendored directory as well which includes Go libraries.

It also saves us having to change into a nested directory:

```sh
https://github.com/redhat-developer/podman-desktop-compose-example
cd compose
podman compose up -d
```

**Review by lstocchi — APPROVED** [review]
LGTM

File: website/blog/2023-11-27-getting-started-with-compose.md
Comment (slemeur) [inline website/blog/2023-11-27-getting-started-with-compose.md]:
```suggestion
[Compose is a specification](https://www.compose-spec.io/) for defining and running multi-container applications. With pose, you use a YAML file to configure your application’s services, networks, and volumes. This allows you to capture in a single file the entire configuration necessary to run a set of interconnected containers as an application. For example, if you have an application that requires a web server, a database, and a caching service, you can define these components and their relationships in your Compose file. Compose is oftenly used in development environment.
```

**Review by slemeur — APPROVED** [review]
Just one minor comment but that would be good to go !
Well done

**Comment by cdrage** [comment]
~DO NOT MERGE until requested by PM. Have other blogs that are in the pipeline (1.6 release) and we should release this blog afterwards.~

Approved!

**Pull request merged**
